docs/en/connector-v2/source/Jdbc.md

```diff
@@ -97,6 +97,9 @@ Source plugin common parameters, please refer to [Source Common Options](common-
 If partition_column is not set, it will run in single concurrency, and if partition_column is set, it will be executed
 in parallel according to the concurrency of tasks.
 
+defaultRowFetchSize param is very import for JDBC Source to read data from Redshift use Redshift driver.
```

Comment:
This part only suite for Redshift, shouldn't put in here. You can add it in example with Redshift.